woocommerce-payments.php

```diff
@@ -38,6 +38,8 @@ function wcpay_activated() {
 		return;
 	}
 
+	// TODO: Can we put something here to track the plugin install?
```

Comment:
We could create the issue for that so we don't forget this one

Reply:
Actually, we don't need it anymore, because we will cover it on Core, but I forgot to remove the comment.